frontend/src/Pages/StudentProfile.tsx

```diff
@@ -0,0 +1,287 @@
+                            useToLocaleString={false}
+                        />
+                        <StatsCard
+                            title="Total Hours"
+                            number={weekNumber}
+                            label={weekLabel}
+                            tooltip={weekToolTip}
+                        />
+                    </div>
+                    {/* Tables */}
+                    <div className="grid grid-cols-2 gap-3 mb-6 mt-6">
+                        <div className="card pt-2 px-3">
+                            <div className="text-teal-4 text-center text-lg font-semibold">
+                                Top 5 Most Viewed Libraries
+                            </div>
+                            <table className="table-2 mb-4">
+                                <thead>
+                                    <tr className="grid-col-3">
+                                        <th className="justify-self-start">
+                                            Library Name
+                                        </th>
+                                        <th># Hours Watching</th>
+                                        <th className="justify-self-end">
+                                            Is Featured
+                                        </th>
+                                    </tr>
+                                </thead>
+                                <tbody className="flex flex-col gap-4 mt-4">
+                                    {metrics.top_libraries.map(
+                                        (items: OpenContentResponse) => {
+                                            return (
+                                                <tr key={items.content_id}>
+                                                    <td className="justify-self-start">
+                                                        <img
+                                                            className="h-8 mx-auto object-contain"
+                                                            src={
+                                                                items.thumbnail_url ??
+                                                                ''
+                                                            }
+                                                        />
+                                                        <ClampedText
+                                                            as="h3"
+                                                            lines={1}
+                                                            className="my-auto w-full body font-normal text-left"
+                                                        >
+                                                            {items.title ??
+                                                                'Untitled'}
+                                                        </ClampedText>
```

Comment:
Can you experiment with either removing the image or reducing the size of it so that the text fits on the line next to the image (and making sure it looks like it feels right)?